tutormfe/patches/openedx-lms-production-settings

```diff
@@ -14,3 +14,42 @@ PROFILE_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% en
 LOGIN_REDIRECT_WHITELIST.append("{{ MFE_HOST }}")
 CORS_ORIGIN_WHITELIST.append("{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}")
 CSRF_TRUSTED_ORIGINS.append("{{ MFE_HOST }}")
+
+# Starts of Dynamic config API settings
+
+ENABLE_MFE_CONFIG_API = True
+
+ALLOWED_HOSTS.append("{{ MFE_HOST }}")
+
+MFE_CONFIG = {
+    "BASE_URL": "{{ MFE_HOST }}",
+    "CREDENTIALS_BASE_URL": "/csrf/api/v1/token",
+    "DISCOVERY_API_BASE_UR": "{% if DISCOVERY_HOST is defined %}{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ DISCOVERY_HOST }}{% endif %}",
+    "ENABLE_NEW_RELIC": False,
+    "FAVICON_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/favicon.ico",
+    "LANGUAGE_PREFERENCE_COOKIE_NAME": "openedx-language-preference",
+    "LMS_BASE_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}",
+    "LOGIN_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/login",
+    "LOGO_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/theming/asset/images/logo.png",
+    "LOGO_TRADEMARK_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/theming/asset/images/logo.png",
+    "LOGOUT_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/logout",
+    "MARKETING_SITE_BASE_URL": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}",
+    "REFRESH_ACCESS_TOKEN_ENDPOINT": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/login_refresh",
+    "SITE_NAME": "{{ PLATFORM_NAME }}",
+    "STUDIO_BASE_URL": "{{ "https" if ENABLE_HTTPS else "http" }}://{{ CMS_HOST }}",
+    "USER_INFO_COOKIE_NAME": "user-info"
+}
+
+
+MFE_CONFIG_OVERRIDES = {
+{% for app in iter_values_named(suffix="MFE_APP") %}
+"{{app.get('name')}}": {
```

Comment:
Also, please add whitespaces: `"{{ app['name'] }}"`

Reply:
While testing for this change I found that it would conflict with readme.rst: 

https://github.com/overhangio/tutor-mfe#adding-custom-translations-to-your-mfes 
> To disable an existing MFE, set its corresponding configuration setting to "null". For instance, to disable the MFEs that ship with this plugin:

> tutor config save --set MFE_ACCOUNT_MFE_APP=null

This will lead to failure, because its null, so should I stick to `.get` or change insturction of disbaling mfe ?